### PR TITLE
Split cudami into two repositories

### PR DESCRIPTION
**Pull request by daforster**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <groupId>de.digitalcollections.cudami</groupId>
```
Comment (daforster) [inline pom.xml:12]:
```suggestion
  <groupId>io.github.dbmdz.cudami</groupId>
```

File: pom.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
   <artifactId>dc-cudami</artifactId>
```
Comment (daforster) [inline pom.xml:13]:
```suggestion
  <artifactId>cudami</artifactId>
```

File: pom.xml [hunk 3]
```diff
@@ -61,157 +61,309 @@
...
+    </dependency>
+    <dependency>
+      <groupId>io.github.dbmdz.metadata</groupId>
+      <artifactId>metasvc-client</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.github.dbmdz.metadata</groupId>
+      <artifactId>metasvc-model</artifactId>
+      <version>${project.version}</version>
```
Comment (daforster) [inline pom.xml:115]:
```suggestion
      <version>9.0.0-SNAPSHOT</version>
```

**Comment by daforster** [comment]
Change package name

**Review by morpheus-87 — APPROVED** [review]

**Review by clorenz — APPROVED** [review]

**Pull request merged**